com.unity.template-hd/Assets/SampleSceneAssets/Scripts/SimpleCameraController.cs

```diff
@@ -188,7 +191,7 @@ void Update()
             // Rotation
             if (IsCameraRotationAllowed())
             {
-                var mouseMovement = GetInputLookRotation() * Time.deltaTime * 5;
+                var mouseMovement = GetInputLookRotation() * Time.deltaTime * 6 * mouseSensitivity;
```

Comment:
I think if you already have an exposed sensitivity factor, the 6 should be part of it.

Reply:
Ok